src/libraries/Microsoft.Extensions.Diagnostics/src/Metrics/DebugConsoleMetricListener.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.Extensions.Diagnostics.Metrics
+{
+    internal sealed class DebugConsoleMetricListener : IMetricsListener, IDisposable
+    {
+        private readonly Timer _timer;
+        internal TextWriter _textWriter = Console.Out;
+        private IObservableInstrumentsSource? _source;
+
+        public DebugConsoleMetricListener()
+        {
+            _timer = new Timer(OnTimer, null, TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(1));
+        }
+
+        public string Name => ConsoleMetrics.DebugListenerName;
+
+        public bool InstrumentPublished(Instrument instrument, out object? userState)
+        {
+            WriteLine($"{instrument.Meter.Name}-{instrument.Name} Started; Description: {instrument.Description}.");
```

Comment:
>instrument.Meter.Name}-{instrument.Name} Star [](http://example.com/codeflow?start=25&length=45)

@noahfalk do we need to write something here regarding meter scope? just in case getting multiple meters with the same name.

Reply:
I'm fine if we don't. I'm hoping it is a distinction most users never need to consider.